vividus-plugin-web-app/src/main/resources/properties/profile/web/profile.properties

```diff
@@ -15,12 +15,19 @@ selenium.grid.url=https://user:accessKey@ondemand.saucelabs.com/wd/hub
 selenium.grid.capabilities.browserName=
 selenium.grid.capabilities.version=
 
+selenium.grid.username=
+selenium.grid.password=
+
+sauceLabs.enabled=false
```

Comment:
@valfirst I propose to add property useLatestVersion from the very start :) It could be very useful

Reply:
Thanks, done